pkg/repository/repository_test.go

```diff
@@ -16,6 +16,7 @@ import (
 	"github.com/go-redis/redismock/v9"
 	"github.com/gofrs/uuid"
 	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/stretchr/testify/require"
```

Comment:
The whole package is tested with `quicktest`, please align with that (`require.NoError(t, err)` -> `c.Assert(err, qt.IsNil)`) if possible.

Reply:
You have also used testify here.
pkg/component/operator/base64/v0/encoding_test.go
https://github.com/instill-ai/pipeline-backend/blob/main/pkg/component/operator/base64/v0/encoding_test.go#L24

Reply:
updated

Reply:
Yes, but:
- I was talking about the `repository` package. We're importing 2 different test packages into the same file. We should avoid that first, then aim to have a single test package in `go.mod`.
- Your reference comes from a package that was in a different repository until the last release.
- Within the `component` repository we had some packages using `testify`, which predate the adoption of `quicktest`. That's the convention now but we didn't refactor the existing coverage. Using `testify` is valid in the packages that already use it (though, if these tests need to be modified, it is desirable to allocate some time to migrate to `qt`).

Reply:
I see. At first I did not learn that quicktest has the checker HaveLen and IsTrue so I used testify, which I am more familiar and I think it's cleaner to write condition checks with it